client/eventsSDL/InputSourceTouch.h

```diff
@@ -78,6 +78,9 @@ struct TouchInputParameters
 	/// gesture will be qualified as pinch if distance between fingers is at least specified here
 	uint32_t pinchSensitivityThreshold = 10;
 
+	/// touch event will trigger clicking of elements up to X pixels away from actual touch position
+	uint32_t touchToleranceDistance = 20;
```

Comment:
Does it make sense to add this param in config (no GUI)?
Maybe one value is better for tablets and another value is better for small smarphones.

Reply:
I guess I can add some param.

Don't see need for custom value for tablets though - current one seems to be quite good on small phones, and should work just fine even on larger screens